File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/Course.java
package com.pennapps.labs.pennmobile;


public class Course {

    private String courseCode;
    private String courseName;
    private String instrName;
    private String avgCourse;
    private String avgInstr;
    private String avgDiff;

    Course(String courseCode, String courseName, String instrName,
           String avgCourse, String avgInstr, String avgDiff) {
        this.courseCode = courseCode;
        this.courseName = courseName;
        this.instrName  = instrName;
        this.avgCourse = avgCourse;
        this.avgInstr = avgInstr;
        this.avgDiff = avgDiff;
    }
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/Course.kt
package com.pennapps.labs.pennmobile.classes


import com.google.gson.annotations.SerializedName

data class Course(@SerializedName("difficulty")
                  val difficulty: Double = 0.0,
                  @SerializedName("course_quality")
                  val courseQuality: Double = 0.0,
                  @SerializedName("num_sections")
                  val numSections: Int = 1,
                  @SerializedName("work_required")
                  val workRequired: Double = 0.0,
                  @SerializedName("description")
                  val description: String = "",
                  @SerializedName("recommendation_score")
                  val recommendationScore: Double = 0.0,
                  @SerializedName("semester")
                  val semester: String = "",
                  @SerializedName("id")
                  val id: String = "",
                  @SerializedName("title")
                  val title: String = "",
                  @SerializedName("instructor_quality")
                  val instructorQuality: Double = 0.0) {
    override fun toString(): String {
        return this.id
    }
}


